_includes/faq/sequential-transactions.md

```diff
@@ -0,0 +1,32 @@
+We find that most use cases that appear to ask for a strong time-based
+write ordering can be solved with other, more distribution-friendly
+solutions instead. For example, CockroachDB's [time travel queries
+(`AS OF SYSTEM
+TIME`)](https://www.cockroachlabs.com/blog/time-travel-queries-select-witty_subtitle-the_future/)
```

Comment:
Once https://github.com/cockroachdb/docs/pull/3018/files lands, we'll have a specific doc page to link to here, which I think it preferable to the blog post.